docs/guide/user_management/user_management.md

```diff
@@ -17,6 +17,41 @@ You can [customize it according to your needs](#customize-login-form).
 The validity of the password recovery token can be set using the `ezplatform.system.<siteaccess>.security.token_interval_spec` parameter.
 By default it is set to `PT1H` (one hour).
 
+### Add forget password option
+
+You can add a "forgot password" option to the site front to allow users of a specific site request a password change.
```

Comment:
I would rather say something along lines, that you can customize template used for `/user/forgot-password` for specific siteaccess, those can be either front or admin ones.